end-to-end-rest/docker-compose.yml

```diff
@@ -9,13 +9,13 @@ services:
       - "9010:8000"
 
   skema-tr:
-    image: enoriega87/skema-webapp
+    image: lumai/askem-skema-text-reading
```

Comment:
@enoriega , should we explicitly add the `:latest` tag as well?  What do you think?

Reply:
Agreed